controllers/cluster_controller_test_test.go

```diff
@@ -161,8 +161,12 @@ func TestClusterReconcilerSetBundlesRef(t *testing.T) {
 }
 
 func newRegistryForDummyProviderReconciler() controllers.ProviderClusterReconcilerRegistry {
```

Comment:
What's the significance of `_test_test` here? are we testing the test file?

Reply:
Historic reasons: `_test.go` used to be in the `controllers` package and `test_test.go` in `controllers_test`. Now that they are both in `controllers_test`, they can be combined. I just didn't do it as part of this PR to keep the scope down.